thirdeye/thirdeye-frontend/app/pods/components/rootcause-heatmap/template.hbs

```diff
@@ -1,17 +1,9 @@
-<button {{action "setMode" "change"}}>change</button> |
-<button {{action "setMode" "contributionDiff"}}>contribution diff</button> |
-<button {{action "setMode" "contributionToDiff"}}>contribution to diff</button>
+urns
+{{#each currentUrns as |urn|}}
+  | <button {{action "setCurrentUrn" urn}}>{{urn}}</button>
```

Comment:
fyi, for straight foward click action that sets a property to a new value, we can use the `mut` helper here and a custom actions is not needed anymore. 
something like: 
```
onclick={{action (mut urn) urn}}
```
https://emberjs.com/api/ember/2.15/classes/Ember.Templates.helpers/methods/mut?anchor=mut

Reply:
yes thanks. fixed